### DISCUSSION TITLE
[FEATURE REQ] Flaresolverr integration

### DISCUSSION THREAD

**Discussion by Flameeeeee** [autobrr/autobrr#1697] [Ideas]
**Is your feature request related to a problem? Please describe.**
Hi, this feature request is about integrating the [Flaresolverr](https://github.com/FlareSolverr/FlareSolverr) solution so it can be used in combination with the `Generic RSS indexer` type to bypass websites sitting behind a CloudFlare protection. 

**Describe the solution you'd like**
I would like to have an option when configuring a `Generic RSS` indexer to specify the address of a FlareSolverr instance that will act as a proxy for every request to the RSS target. It could also be implemented in a different way like it is done in Sonarr, Radarr, Prowlarr ... With a single instance configured and tags applied to each indexer needing this proxy.
![1](https://github.com/autobrr/autobrr/assets/54583513/f77b8487-6bc7-44a7-810a-b511be946768)
Prowlarr implementation: 
![2](https://github.com/autobrr/autobrr/assets/54583513/2573a6ba-50ad-434f-bf04-6ecd538ea561)

**Describe alternatives you've considered**
I am currently using the `Generic Torznab` indexer which points to my Prowlarr instance which benefits from a Flaresolverr integration. However this solution is not ideal since it relies on the search engine of the indexer which takes like 2 hours to get updated with new releases compared to the RSS.

**Additional context**
I was using the `Generic RSS` indexer type for one of my indexers a few months ago but after some DDoS attacks this indexer raised his CloudFlare protection level to a point where even the RSS page is protected and inaccessible from automated systems like Autobrr or any other crawler.

**Comment by zze0s** [comment]
Hey! Looks like you're not the only one who want this feature.

We've been planning on adding proxy support for indexer/feed downloads for quite a while so this should probably be part of that.

**Comment by bilyboy785** [comment]
Hello @zze0s do you have any ETA for this feature ?

**Comment by nihalxx3** [comment]
Did you find any solution or able to update autobrr to use flaresolverr ?
